tests/map/options.test.ts

```diff
@@ -30,4 +31,33 @@ describe('MapOptions', () => {
         assert(document.body.innerHTML === '');
         assert(root.innerHTML === '');
     });
+
+    it('scale label for min/max switches between linear and log', () => {
+        const root = document.createElement('div');
+        const options = new MapOptions(root, DUMMY_PROPERTIES, DUMMY_CALLBACK);
+
+        assertScaleLabel(options.x, 'x');
+        assertScaleLabel(options.y, 'y');
+        assertScaleLabel(options.z, 'z');
+
+        function assertScaleLabel(axisOptions: AxisOptions, axisName: string): void {
+            const min = getByID(`chsp-${axisName}-min-label`);
+            const max = getByID(`chsp-${axisName}-max-label`);
+            const selectElement = getByID<HTMLSelectElement>(`chsp-${axisName}-scale`);
+
+            // change from linear (default) to log scale
+            selectElement.selectedIndex = 1;
```

Comment:
why not use

```suggestion
            selectElement.value = "log";
```

Reply:
I didn't know that's an option, thanks :D